### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by TomfromBerlin**
As the title says, the German translation for Windhawk. I made this with great care and hope it helps Windhawk to become more popular.

### CODE REVIEW

File: de.yml [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      contribute: "<0>Eine neue Übersetzung erstellen</0>."
+      credits: "Deutsche Übersetzung: <0>TomfromBerlin</0>."
+      creditsLink: "mailto:tom-at-github@quantentunnel.de"
+    updates:
+      title: "Nach Aktualisierungen suchen"
+      description: "Regelmäßig nach neuen Versionen von Windhawk und den installierten Mods suchen"
+    devMode:
+      title: "Entwickler-Modus"
+      description: "Aktionen für Entwickler anzeigen, z. B. das Erstellen und Ändern von Mods"
+    advancedSettings: "Erweiterte Einstellungen
```
Comment (m417z) [inline de.yml:191]: A quote is missing here.

File: de.yml [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      arrayItemRemove: "Element entfernen"
+    code:
+      title: "Quellcode"
+      noData: "Mod-Quellcode fehlt."
+      collapseExtra: "Readme und Einstellungen einklappen"
+    changelog:
+      title: "Änderungsprotokoll (Changelog)"
+      loadingFailed: "Laden des Änderungsprotokolls fehlgeschlagen. <0>Hier klicken</0> um das Änderungsprotokoll auf GitHub zu lesen."
+    advanced:
+      title: "erweitert"
```
Comment (m417z) [inline de.yml:132]: This probably should be capitalized.

**Review by m417z — COMMENTED** [review]
Thank you very much!

File: de.yml (hunk 2 above)
Comment (TomfromBerlin) [reply de.yml]: Done. Nice app that is, btw.

File: de.yml (hunk 1 above)
Comment (TomfromBerlin) [reply de.yml]: Quote added.

**Pull request merged**